app/src/main/java/org/andstatus/todoagenda/prefs/HideSubtasks.java

```diff
@@ -0,0 +1,34 @@
+package org.andstatus.todoagenda.prefs;
+
+import androidx.annoation.StringRes;
```

Comment:
Hi @Zocker1999NET Thanks for the contribution.
The strange thing is that your code has lots of compilation errors :-)
Do you use Android Studio for development? I mean it shows all errors and it's easy to fix them...

Reply:
No :), yesterday I wasn't home while coding, so I wrote the first version in VS Code without any Android/Java plugins applied (otherwise my laptop battery would have been died way too quick). That's why I haven't checked for compile errors for now xD.
I thought I marked this as draft and not as a "ready to merge" PR, but either I missed something or GitHub forgot that … It is not ready to merge yet, I just wanted that you and other collaborators can check on the progress and maybe hint on conceptional errors as early as possible.

Reply:
As I see, you do most of additions using analogy with adjacent code, so everything is OK so far.
You didn't do all necessary additions for your first step yet, I think,
and this means that adding the new setting to existing tests will allow to show this.

So please don't postpone tests addition to the next implementation step.